packages/demo/src/components/examples/RadioExamples.tsx

```diff
@@ -138,3 +146,43 @@ const RadioSelectWrappedExample: React.FunctionComponent = () => (
         </RadioSelectConnected>
     </Section>
 );
+
+const RadioCardSelectExample: React.FunctionComponent = () => (
+    <Section level={3} title="A radio card select.">
```

Comment:
level={2} to be consistent with the others and remove the dot at the end of the title. (and at the same time, can you do the same thing for the section over this one (line 128, `<Section level={3} title="A radio select with a wrapped radio button.">`, level 2 and remove the dot in the title. Thanks!